src/style.css

```diff
@@ -0,0 +1,75 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+* {
+  margin: 0;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+}
+
+.nav-menu {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+  margin: 0;
+}
+
+.item-link {
+  position: relative;
+  display: block;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
```

Comment:
why you specify font-style here?

Reply:
This was copied from figma mockup

Reply:
no need to copy everything from figma, use only that styles that is really necessary for the page and don't forget that some styles are default therefore you don't need to mentioned them again